gen/flights/closestairfields.py

```diff
@@ -25,7 +25,11 @@ def __init__(
 
     @property
     def operational_airfields(self) -> Iterator[ControlPoint]:
-        return (c for c in self.closest_airfields if c.runway_is_operational())
+        return (
+            c
+            for c in self.closest_airfields
+            if c.runway_is_operational() or c.has_helipads
```

Comment:
Do you want helipads to be an OCA target? If so you probably want to give them their own `RunwayStatus` and factor out a common `c.airfield_is_operational`

Reply:
I don't think it'll be necessary in a first step to make pads destroyable. I don't think they can be destroyed in DCS anyway.
For now i'm also not yet generating the unused units, so OCA/Aircraft is not a thing.

This bit of code was mainly to allow planning flights from the FARP/FOB. And I refactored this in the Fob class.

Note : Shouldn't we allow helicopter flights from airbases, even when the runway is not operational ?